octoprint_slic3r/__init__.py

```diff
@@ -288,9 +288,19 @@ def do_slice(self, model_path, printer_profile, machinecode_path=None, profile_p
       import subprocess
 
       help_process = subprocess.Popen((executable, '--help'), stdout=subprocess.PIPE)
-      help_text = help_process.communicate()[0]
+      help_text_all = help_process.communicate()
+      
+      # help output includes a trace statement now on the first line. If we find it, use the second
+      # line instead
+      # [2022-04-22 21:44:51.396082] [0x75527010] [trace]   Initializing StaticPrintConfigs
+      if help_text_all[0].find(b'trace') >= 0
```

Comment:
I think that you could also have written:

`if b'trace' in help_text_all[0]`.

Either way!